sdks/python/apache_beam/io/gcp/bigquery_write_it_test.py

```diff
@@ -368,9 +376,9 @@ def test_big_query_write_temp_table_append_schema_update(self):
     table_id = '{}.{}'.format(self.dataset_id, table_name)
 
     input_data = [{
-        "int64": num, "bool": True, "nested_field": {
+        "int64": num, "bool": True, "nested_field": [{
             "fruit": "Apple"
-        }
+        }]
```

Comment:
😮  wow how did this test pass before?

Reply:
it used to be a JSON format only. JSON writer is less restrictive on that case. When I've added AVRO then found that `fastavro` requires dicts be surrounded by list

https://github.com/fastavro/fastavro/issues/254#issuecomment-819038448